src/assets/components/button.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+function Button({ children, clickHandler, isDisabled }) {
```

Comment:
Deze lijkt niet gebruikt te worden

Reply:
dit had ik idd in het begin gemaakt, wil ik graag components maken maar had ik niet genoeg niet tijd op het eind. Ik had dit moeten uit laten.